### ISSUE TITLE
i have no idea how to download it 

### ISSUE THREAD

**Issue by samuel823** [Ice3man543/subfinder#55]
u should make a video or something , ur explain is so bad , im sure no one can understand it .....

**Comment by samuel823** [comment]
look , in the Docker Container part, i was like what was it

**Comment by Ice3man543** [comment]
What are you facing problem with?

**Comment by samuel823** [comment]
i just dont know what to do after i download subfinder , becuase i didnt see the subfinder there , so that the mean is i cant just run ./subfinder -d (target)

**Comment by Ice3man543** [comment]
What OS are you using? How are you installing it? Provide the exact details!

**Comment by samuel823** [comment]
i got linux , the only problem is i cant run the tools with ./subfinder -d
target.com < becuase there have no subfinder dude

**Comment by Ice3man543** [comment]
Do you know what gopath is? Have you installed golang?

**Comment by Ice3man543** [comment]
Try to read about golang and how to install it and write programs in it. :+1:

**Comment by samuel823** [comment]
yes , but no i didnt

**Comment by samuel823** [comment]
i mean can u just give me the command like after i download golang , what
is the command with run subfinder

**Comment by samuel823** [comment]
ty so much bro

**Comment by samuel823** [comment]
no , i know how to download golang , dude all i need to know is the command
about run the tools

**Comment by samuel823** [comment]
becuase the explain u put in the page is 100% wrong and now im asking

**Comment by samuel823** [comment]
i mean codingo not u , and second maybe u not even better than me , so yea if u dont know anything just shut up pls...

**Comment by Ice3man543** [comment]
@samuel823 The tool repository you're currently viewing, i am the lead author of the tool. It was my idea, i worked on the major parts, created the passive engine, written 16 sources, resolver,bruteforcer, etc all by myself. And now you, who doesn't even knows how to install and run a simple golang tool, who doesn't even knows anything about golang or anything about computers says to me that i don't know more than you.

So what I suggest is that you create your own subfinder. Or any such security tool. And then I'll see what you are.

As far as my explanation is concerned, it's understandable and pretty legible for people who know what they are doing. I didn't intend my tool to be used by **utter noobs** and people who have completely no idea of what they are doing but be arrogant as they know everything. So the person who should better shup the hell up is you. Better control your anger. 

If you can't show respect, get away from here. Do you even know what git is?
I have made a good tool and everyone knows that. And you're not paying me for making this. So, i am not responsible for making sure that every single noob understands how to run this tool.
Thanks
- @Ice3man (The author and guy with bad explanation)

**Comment by samuel823** [comment]
dude are u like stupid in real life or someshjt i just need the fcking command about run the tools , i know how to donwload it , and im sure im better than u with hacking dumb ass

**Comment by Ice3man543** [comment]
@samuel823 You are a complete idiot. Can't you even see i am the owner of this repo. As far as my hacking is concerned, show me one achievement you got? I ain't gonna waste my time with scumbags like you. What do you know man? A person who can't install a simple tool is much more stupid than a person who has written that tool.

Why am i bothering myself. You are a noob and idiot and will remain so in the future. If you have the guts, other than swearing create a similar tool and then we'll see who outruns whom.

Your'e a **stupid**, **arrogant**, **idiot**, and **noob** person.
Learn some shit and then come talk to me :-1:

**Comment by samuel823** [comment]
hahahahaha someone got mad

**Comment by samuel823** [comment]
trashhhhhhhhhhhh

**Comment by Ice3man543** [comment]
Yeah, unfortunately, there are people like you in this world who want results fed into their mouth without doing anything and trying to learn anything. And i really don't wanna waste my time on people like you, so Peace 

- @Ice3man

**Comment by samuel823** [comment]
hahahahahah no one care about ur shjt tools and u cant even find a vuln in
any website hahahaha , trashhhhhhh

**Comment by Ice3man543** [comment]
Hi,
Oh, really. If that's the case then what are you doing here? 
As far as my vulnerability hunting is concerned, i have found bugs in programs. 
I am not really like you to go and run `inurl:admin.php` into google and use **1'or'1'='1** without even knowing what it means.
Thanks for your concern.

**Comment by samuel823** [comment]
sorry im doing google hacking but not just like inurl:php?id= and getting
subdomain with some good tools not ur shjt tools and then web discover and
after using burp suite to check . haahahahah  im sure u dont even know what
burp suite is dumb ass lollllllllllllllllllll

**Comment by samuel823** [comment]
im sure u just stay home making trash tools and no one care :D , that ur
life hahahah

**Comment by Ice3man543** [comment]
@samuel823 Burp suite - the tool with the name of some guy called Larry Lau on it's title window?

**Comment by zer0trip** [comment]
skids everywhere man

**Comment by jhaddix** [comment]
This thread hurts my soul,

**Comment by haxormad** [comment]
Damn this guy. He don't even know who ice3man and codingo are and telling himself elite lol. Skids these days

**Comment by CaptainFreak** [comment]
Oh this is LOL :)))))
